datamodel/ResultsAggregator.py

```diff
@@ -226,6 +253,12 @@ def get_case_metadata_xml(case, filename):
         if _info and len(_info) >= 3:
             _meta['priority'] = _info[0]
             _meta['severity'] = _info[1]
-            _meta['squad(s)'] = _info[2]
+            _meta['squad(s)'] = _info[2].split(',')
+        else:
+            # Metadata not set - set defaults
+            _meta['priority'] = "Priority/P1"
+            _meta['severity'] = "Severity 1 - Urgent"
```

Comment:
Sev 1 is the _default_?  This seems really aggressive to me.

Reply:
If I recall correctly this was something PM chimed in on - but we can get some input from @anthonyfister / @chuckbrant / @blkingrh on it!  

Folks - what should the default priority/severity be for a squad's test if they don't specify a priority/severity in the metadata?

Reply:
I say bring the hammer down if we can't get proper information on a test we treat it as critical and put the onus on the squad to triage it as needed until they either add metadata to give us importance or remove it as not necessary enough to maintain.